app/(pages)/_components/products/_components/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styles from './ProductCard.module.scss';
+import {
+  FaCheckCircle,
+  FaTimesCircle,
+  FaRegTrashAlt,
+  FaPencilAlt,
+} from 'react-icons/fa';
+
+// Still unsure about this part since the figma online doesn't have a listed boolean, so for now, just a placeholder
+interface ProductInfo {
+  imageUrl: string;
+  title: string;
+  listed: boolean;
+}
+
+interface ProductCardProps {
+  productInfo: ProductInfo;
+}
+
+const ProductCard: React.FC<ProductCardProps> = ({ productInfo }) => {
+  return (
+    <div className={styles.productCard}>
+      <img
```

Comment:
Please use the Image component from Next.js instead of img for some optimizations!